pkgs/tools/inputmethods/ibus-engines/ibus-mozc/default.nix

```
@@ -1,5 +1,6 @@
-{ clangStdenv, fetchFromGitHub, fetchpatch, which, ninja, python, gyp, pkgconfig
-, protobuf, ibus, gtk2, zinnia, qt5, libxcb, tegaki-zinnia-japanese }:
+{ clangStdenv, fetchFromGitHub, which, ninja, python, gyp, pkgconfig
+, protobuf, ibus, gtk2, zinnia, qt5, libxcb, tegaki-zinnia-japanese, python37Packages
```

```suggestion
, protobuf, ibus, gtk2, zinnia, qt5, libxcb, tegaki-zinnia-japanese, python3Packages
```
Please don't hardcode minor python versions.